src/app/frontend/replicasetdetail/replicasetdetail.html

```diff
@@ -63,7 +63,7 @@ <h1 flex class="md-title kd-replicasetdetail-app-name">
             </md-button>
           </div>
         </div>
-        <span class="kd-replicasetdetail-sidebar-line">Pods status</span>
+        <span class="kd-replicasetdetail-sidebar-line">Pods Status</span>
```

Comment:
Hey, I think this is the opposite. The previous version is correct. I think Ken wanted to change labels on the cards list, which I already done. Ref: https://msdn.microsoft.com/en-us/library/windows/desktop/bb246428(v=vs.85).aspx